### ISSUE TITLE
❌ Missing Information: Lightbox title and description not working properly

### ISSUE THREAD

**Issue by mickfollari** [elementor/elementor#20040]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

On my client's site, we're using the Elementor lightbox. For some reason in the lightbox, the image description is in the title div (the elementor-slideshow__title div), and the description tag div is empty. So we get no title and only the description appearing where the title should be.
screenshot of site and inspector:
https://www.screencast.com/t/0fBF9mmvg3HG

Example page: https://erindonnelly.com/work/abstract/

### Steps to reproduce

Go to the site, open one of the image gallery pages (mentioned above).
Click on one of the images in the grid.
Notice the problem in the lightbox. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.6.51
	PHP Version: 8.0.23
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 32M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.2
	Site URL: https://erindonnelly.com
	Home URL: https://erindonnelly.com
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/Denver
	Admin Email: erinoes@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: ErinD2022
	Version: 1.0.0
	Author: Mick Follari
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 3.8.5
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:105.0) Gecko/20100101 Firefox/105.0

== Active Plugins ==
	Contact Form 7
		Version: 5.6
		Author: Takayuki Miyoshi

	Elementor
		Version: 3.6.6
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 5.1.7
		Author: WPDeveloper

	Jetpack
		Version: 11.0
		Author: Automattic

	Premium Addons for Elementor
		Version: 4.9.20
		Author: Leap13

	Regenerate Thumbnails
		Version: 3.1.5
		Author: Alex Mills (Viper007Bond)

	UpdraftPlus - Backup/Restore
		Version: 1.22.12
		Author: UpdraftPlus.Com, DavidAnderson

	WooCommerce
		Version: 6.6.1
		Author: Automattic

	WooCommerce Payments
		Version: 4.3.1
		Author: Automattic

	WooCommerce Shipping & Tax
		Version: 1.26.1
		Author: WooCommerce

	Wordfence Security
		Version: 7.7.1
		Author: Wordfence

== Elements Usage ==
	
	page : 2
		premium-img-gallery : 1
		sidebar : 2
		spacer : 1
	wp-page : 12
		eael-contact-form-7 : 1
		eicon-woocommerce : 1
		heading : 13
		image : 11
		premium-img-gallery : 8
		sidebar : 8
		spacer : 10
		text-editor : 3

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Inactive by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default

== Log ==
	
JS: showing 1 of 1JS: 2022-04-28 23:39:09 [error X 3][http://erindonnelly.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.5:2:515477] T is null 

PHP: showing 1 of 1PHP: 2022-05-10 23:01:43 [warning X 3][/home/illa4mixkwj9/public_html/wp-content/plugins/elementor/includes/managers/image.php::87] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 9 of 92022-06-29 11:32:06 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.6.6',
)]
2022-06-29 11:32:07 [info] elementor::elementor_updater Started 
2022-06-29 11:32:07 [info] Elementor/Upgrades - _on_each_version Start  
2022-06-29 11:32:07 [info] Elementor/Upgrades - _on_each_version Finished 
2022-06-29 11:32:08 [info] elementor::elementor_updater Started 
2022-06-29 11:32:08 [info] Elementor/Upgrades - _on_each_version Start  
2022-06-29 11:32:08 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.6.6',
)]
2022-06-29 11:32:08 [info] Elementor/Upgrades - _on_each_version Finished 
2022-06-29 11:32:08 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.5',
  'to' => '3.6.6',
)]

== Elementor - Compatibility Tag ==
	
	Essential Addons for Elementor: Compatibility not specified
	Premium Addons for Elementor: Compatibility not specified

**Comment by mickfollari** [comment]
Edit: I updated everything today, and now the Title is correct in the lightbox, but the description is empty. I tried setting it to 'description' field and the 'caption' field to test and neither is populating the field in the lightbox.

**Comment by nicholaszein** [comment]
Hello @mickfollari! 

Thank you for submitting your issue! 🙏 

Can you please confirm the questions below?

1. Can you confirm this happens with only Elementor active?
2. Can you confirm this happens with only the Hello Elementor theme active?
3. Can you confirm your images have Captions, Titles, and Descriptions, and that they are different?
4. Can you confirm this happens with the native Basic Gallery widget from Elementor, instead of the Premium Addons Gallery widget you're currently using?

I also see that the two Elementor add-ons you're using are incompatible with the current version of Elementor installed on your website. Elementor add-ons can modify Elementor features and this might lead to conflicting issues.

It's also worth mentioning that although you marked that this issue happens with the latest stable version of Elementor installed, I see in your System Info that you have Elementor v3.6.6 installed, an older version of Elementor.

👉 Please follow the guidelines for opening a [new issue report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

⚠️ This report is missing key information: 

* **Prerequisites:** You need to make sure you have tested with the latest version of Elementor installed.
* **Isolating the problem:** You need to make sure you have isolated the problem by testing with **only** Elementor active.

Without them, we simply can't help. 😕 

👍 Please feel free to open a new issue in accordance with these guidelines.

Best regards,

Nick

**Comment by detinaa** [comment]
Hello, I have the same issue with Lightbox title and description not working
